geoshape/README.md

```diff
@@ -0,0 +1,31 @@
+## Geopoint track
+
+This track is based on [PlanetOSM](http://wiki.openstreetmap.org/wiki/Planet.osm) data.
+
+### Example Document
+
+```json
+{
```

Comment:
Sure. The script basically takes the file that @nknize provided in https://github.com/elastic/rally-tracks/issues/60#issuecomment-449483186 and converts it into a single element JSON, so we are unlikely to run it ever again. But I will add it just in case.